src/block-management/block-manager.js

```diff
@@ -201,6 +226,7 @@ export class BlockManager extends React.Component<PropsType, StateType> {
 					keyboardShouldPersistTaps="always"
 					style={ styles.list }
 					data={ this.props.blockClientIds }
+					extraData={ [ this.state.isTitleFocused, this.state.isFullyBordered ] }
```

Comment:
I don't see this property being used in `KeyboardAwareFlatList` or somewhere else. What is it for? 🤔

Reply:
By passing extraData we make sure FlatList will re-render when extraData changes, otherwise changes won't get effect. If you look inside `keyboard-aware-flat-list.ios.js` or `keyboard-aware-flat-list.android.js` you can see all props are propagated to react-native's FlatList.

Reply:
Perfect! Thank you.